ideas/308-documentation.md

```diff
@@ -0,0 +1,133 @@
+---
+id: 308-documentation
+title: Documentation
+status: draft
+created: 2018-09-26
+category: multi-category
+lead-contributor: andytudhope (Andy Tudhope)
+contributors:
+    - easye
+    - rachelhamlin
+    - jeluard
+    - hazel
+    - Jakub
+exit-criteria: no
+success-metrics: 
+    - re-styled according to Hazel's designs to match web-refresh
+    - global header navigation to move across all Status domains
+    - clear documentation that is easy to contribute to and manage
+    - kept up-to-date month on month, with at least 5 contributions and 2 tutorials
+    - clear user journeys established once content is flowing
+clear-roles: no
+future-iteration: yes
+roles-needed:
```

Comment:
Great stuff, maybe we can briefly mention this on core dev call and get people to commit from each relevant team?

I'd say maybe instead of Go/Clojure team (or perhaps in addition to it) we should ensure areas such as infra/p2p/security; mobile/desktop; chat/wallet/dapps are covered.